lib/manager/mix/extract.ts

```diff
@@ -67,7 +67,8 @@ export async function extractPackageFile(
     }
   }
   const res: PackageFile = { deps };
-  const lockFileName = getSiblingFileName(fileName, 'mix.lock');
+  const lockFileName =
+    (await findLocalSiblingOrParent(fileName, 'mix.lock')) || 'mix.lock';
```

Comment:
Why the fallback

Reply:
It needs to be a string, otherwise it causes an exception if there is no lockfile at all. There is a test case covering this which failed on my first commit